frontend/src/metabase/visualizations/lib/columns.ts

```diff
@@ -0,0 +1,27 @@
+import { t } from "ttag";
+
+import Table from "metabase/visualizations/visualizations/Table";
```

Comment:
Was a bit worried about a cyclic dependency here, but it seems to work fine

Reply:
We could also define `getTitleForColumn` in `Table.jsx` and export it from there, that would avoid the cyclic dependency.

Reply:
For sure, and that may be the better way to go here. We tend to tuck this kinda stuff into lib or util files outside of the components, but all the static class props make this a little awkward 🥲

Reply:
Yep, let's extract this function to avoid having cyclic dependencies